framework/src/application/application.js

```diff
@@ -538,10 +526,6 @@ class Application {
 				getBlockByHeight: {
 					handler: async action => this._node.actions.getBlockByHeight(action),
 				},
-				getBlocksByHeights: {
```

Comment:
why remove this?

Reply:
it was unused and there was no action present as well.. we have `getBlockByHeight` and `getBlockByHeightBetween` which serve the purpose. this code was a bug we left i guess.